x/concentrated-liquidity/tick.go

```diff
@@ -36,19 +36,6 @@ func (k Keeper) initOrUpdateTick(ctx sdk.Context, poolId uint64, currentTick int
 	// calculate liquidityGross, which does not care about whether liquidityIn is positive or negative
 	liquidityBefore := tickInfo.LiquidityGross
 
-	// if given tickIndex is LTE to the current tick and the liquidityBefore is zero,
-	// set the tick's spread reward growth opposite direction of last traversal to the spread factor accumulator's value
-	if liquidityBefore.IsZero() {
-		if tickIndex <= currentTick {
-			accum, err := k.GetSpreadRewardAccumulator(ctx, poolId)
-			if err != nil {
-				return false, err
-			}
-
-			tickInfo.SpreadRewardGrowthOppositeDirectionOfLastTraversal = accum.GetValue()
-		}
-	}
-
```

Comment:
Note that this is already achieved by the call to this:
https://github.com/osmosis-labs/osmosis/blob/0f654ff4f02f328cf46265606b8e48576021554f/x/concentrated-liquidity/tick.go#L122

Reply:
Oh wow was this setup completely duplicate? In any case, having `currentTick` as a param seems like it was unnecessary since all CL calls to it just seem to pass in `pool.GetCurrentTick()`. We should be careful with this but it looks like a good cleanup